framework/helper/randomisation.h

```diff
@@ -0,0 +1,14 @@
+#ifndef T3_RANDOMISATION_H
+#define T3_RANDOMISATION_H
+
+void T3_Init_Random();                                      //Initialize the random generator seed (time)
+
+bool T3_Random_Bool();                                      //Generate a boolean random (returns true or false)
```

Comment:
if you use this syntax above the method

```
/**
* Some explanation
*/ 
```
it also shows your explanation anywhere the method called. so maybe you can change the comments to this format?

Reply:
Changed the comments syntax.

Thanks again, I did not know you can achieve that.